### PR TITLE
Breaking: Making it compatible with HA 0.116

### PR DESCRIPTION
**Pull request by noxhirsch**
This makes the multiple entity row compatible with HA 0.116 and resolves issue #103 but is a breaking change (so <0.115 users can't upgrade).
More information here https://youtu.be/6Oi__1yuL_s?t=703

### CODE REVIEW

File: multiple-entity-row.js [hunk 1]
```diff
@@ -315,4 +315,4 @@
     }
 
     customElements.define('multiple-entity-row', MultipleEntityRow);
-})(window.LitElement || Object.getPrototypeOf(customElements.get('hui-view')));
+})(window.LitElement || Object.getPrototypeOf(customElements.get('hui-masonry-view')));
```
Comment (firstof9) [inline multiple-entity-row.js:318]: Would this work to keep < 0.116 working?

```
customElements.define("multiple-entity-row", MultipleEntityRow);
})(
  window.LitElement || customElements.get("hui-masonry-view")
    ? Object.getPrototypeOf(customElements.get("hui-masonry-view"))
    : Object.getPrototypeOf(customElements.get("hui-view"))
);
```

Comment (benct) [reply multiple-entity-row.js:318]: It should. Can be simplified with the following:
```suggestion
})(window.LitElement || Object.getPrototypeOf(customElements.get('hui-masonry-view')) || Object.getPrototypeOf(customElements.get('hui-view')));
```

Comment (noxhirsch) [reply multiple-entity-row.js:318]: The solution by @firstof9 works in HA <0.116 and 0.116

@benct This gives an error in HA <0.116 (but works in 0.116):
```
Uncaught TypeError: Cannot convert undefined or null to object
    at Function.getPrototypeOf (<anonymous>)
    at multiple-entity-row.js:318
```

Comment (benct) [reply multiple-entity-row.js:318]: Sorry, try this instead..
```suggestion
})(window.LitElement || Object.getPrototypeOf(customElements.get('hui-masonry-view') || customElements.get('hui-view')));
```

Comment (noxhirsch) [reply multiple-entity-row.js]: Now it's working fine for 0.115 & 0.116

**Review by firstof9 — COMMENTED** [review]
Changes look good to my js noob eyes.

**Pull request merged**